ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -484,6 +490,10 @@ textFunctionBase
     | RIGHT | LEFT | ASCII | LOCATE | REPLACE
     ;
 
+    positionFunctionName
+        : POSITION
+        ;
```

Comment:
Can you fix indents?

Reply:
Fixed in 415e1636add906b2dca1317ecbbf4098269426ba